nix/default.nix

```diff
@@ -14,14 +14,36 @@
 #  limitations under the License.
 #
 
-{ system ? builtins.currentSystem
-, pkgs ? import <nixpkgs> { inherit system; }
+let
+  nixpkgs-bootstrap = import <nixpkgs> { };
+
+  nixpkgs-16_03 = import (nixpkgs-bootstrap.fetchFromGitHub {
+    owner = "NixOS";
+    repo = "nixpkgs-channels";
+    rev = "baf46b99e33005348fdbd083366c330be4b373f3";
```

Comment:
Is there any possibility to set a pointer to the release branch? It looks like that `rev` points to a specific commit in the development branch. For me, It is unclear, why this comment has been chosen.